src/rust/lib/graph-editor/src/component/visualization/renderer/sample.rs

```diff
@@ -0,0 +1,5 @@
+//! The `sample` modules contains some examples of visualisations.
```

Comment:
Rename the module to `examples`. According to Ara, `sample` is much less formal and should not be used in codebase name